packages/sit-onyx/src/i18n/locales/en-US.json

```
@@ -4,6 +4,13 @@
   "apply": "Apply",
   "back": "Back",
   "close": "Close",
+  "data-grid": {
+    "head": {
+      "sorting-action-asc": "Click to sort the table ascending by the {field} column.",
+      "sorting-action-desc": "Click to sort the table descending by the {field} column.",
+      "sorting-action-none": "Click to reset the sorting order."
```

```suggestion
      "sorting-action-asc": "Sort the table ascending by the {field} column.",
      "sorting-action-desc": "Sort the table descending by the {field} column.",
      "sorting-action-none": "Reset the sorting order."
```

The action can also be triggered via keyboard, right? So technically "Click to" is not correct